waveshare-epd/epd1in54/epd1in54.go

```diff
@@ -79,22 +82,28 @@ var partialRefresh = [159]uint8{
 }
 
 // New returns a new epd1in54 driver. Pass in a fully configured SPI bus.
-func New(bus *machine.SPI, csPin, dcPin, rstPin, busyPin machine.Pin) Device {
+func New(bus *machine.SPI, csPin, dcPin, rstPin legacy.PinOutput, busyPin legacy.PinInput) Device {
 	return Device{
 		buffer: make([]uint8, (uint32(Width)*uint32(Height))/8),
 		bus:    bus,
-		cs:     csPin,
-		dc:     dcPin,
-		rst:    rstPin,
-		busy:   busyPin,
+		cs:     csPin.Set,
+		dc:     dcPin.Set,
+		rst:    rstPin.Set,
+		busy:   busyPin.Get,
```

Comment:
```suggestion
		isBusy:   busyPin.Get,
```
... as one example for an input-pin function, see also my remarks for the Set-function (apa102/softspi.go), the same applies for the usage of legacy-Set()-function

Reply:
I do agree isBusy is much better than busy! Changed in latest commit!